crates/bevy_core_pipeline/Cargo.toml

```diff
@@ -47,7 +47,6 @@ nonmax = "0.5"
 smallvec = "1"
 thiserror = { version = "2", default-features = false }
 tracing = { version = "0.1", default-features = false, features = ["std"] }
-bytemuck = { version = "1" }
```

Comment:
byte muck used to be needed to be in scope but unused for some macro. is that not the case anymore?